tests/supertokens_python/test_supertokens_functions.py

```diff
@@ -71,6 +73,11 @@ async def test_supertokens_functions():
     emails_desc = [user.email for user in users_desc]
     assert emails_desc == emails[::-1]
 
+    version = await Querier.get_instance().get_api_version()
+    if not is_version_gte(version, "2.10"):
+        # If the version less than 2.10, delete user feature didn't exist, so skip the test
+        skip()
```

Comment:
Instead of skipping, you should do a return - cause you are still doing the above checks

Reply:
done.